app/src/main/res/values-in/strings.xml

```diff
@@ -162,38 +168,38 @@
     <string name="request_timeout">Batas waktu permintaan</string>
     <string name="superuser_notification">Notifikasi superuser</string>
     <string name="settings_su_reauth_title">Autentikasi ulang setelah upgrade</string>
-    <string name="settings_su_reauth_summary">Autentikasi ulang izin akses superuser setelah aplikasi diupgrade</string>
+    <string name="settings_su_reauth_summary">Autentikasi ulang izin akses superuser setelah mengupgrade suatu aplikasi</string>
     <string name="settings_su_tapjack_title">Aktifkan perlindungan tapjacking</string>
     <string name="settings_su_tapjack_summary">Dialog permintaan superuser tidak akan menanggapi masukan saat terhalangi oleh lapisan atau jendela lainnya</string>
     <string name="settings_su_biometric_title">Aktifkan autentikasi biometrik</string>
     <string name="settings_su_biometric_summary">Gunakan autentikasi biometrik untuk mengizinkan permintaan superuser</string>
     <string name="no_biometric">Perangkat tidak mendukung atau setelan biometrik tidak diaktifkan</string>
     <string name="settings_customization">Personalisasi</string>
-    <string name="setting_add_shortcut_summary">Tambahkan pintasan yang menarik pada layar utama seandainya nama dan ikon sulit untuk dikenali setelah menyembunyikan aplikasi</string>
+    <string name="setting_add_shortcut_summary">Tambahkan pintasan yang menarik di layar utama seandainya setelah menyembunyikan aplikasi nama dan ikon sulit untuk dikenali</string>
     <string name="settings_doh_title">DNS melalui HTTPS</string>
-    <string name="settings_doh_description">Solusi untuk DNS poisoning di beberapa negara</string>
+    <string name="settings_doh_description">Solusi untuk permasalahan DNS poisoning di beberapa negara</string>
```

Comment:
String change not necessary here, unless the original string also changes to have it read "... DNS poisoning issue(s) ..."